actions/pack_action.go

```diff
@@ -35,5 +35,7 @@ func (pf *PackAction) Run(context *debos.DebosContext) error {
 	outfile := path.Join(context.Artifactdir, pf.File)
 
 	log.Printf("Compressing to %s\n", outfile)
-	return debos.Command{}.Run("Packing", "tar", "czf", outfile, "-C", context.Rootdir, ".")
+	return debos.Command{}.Run("Packing", "tar", "czf", outfile,
+		"--xattrs", "--xattrs-include=security.capability",
```

Comment:
should it really only include security.capability? I can imagine recipes creating others as well.

Reply:
I initially saved all the xattrs, because why not.

Then it happened that I'm running with SELinux enabled and then all the SELinux labels from my system were captured as well, which is rather undesirable.

So I decided to play safe and include only the thing I specifically care about in this round. I suppose some configurability would be needed for more complicated use-cases, but I tried to stay away from that rabbit hole.

Reply:
That's surprising as debos will build in either RAM on the VM or an extra image (scratch image) ; i wonder what's adding the selinux labels?

Did you test without fakemachine maybe?

Reply:
IIRC I did some tests with `go test` and that's probably where the xattrs got mixed up. They indeed don't appear when using fakemachine from, say, the Apertis container.

Reply:
Updated the MR to use `*.*` and save all xattrs.